targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_KL27Z/device/TOOLCHAIN_ARM_STD/MKL27Z64xxx4.sct

```diff
@@ -51,7 +51,7 @@
 #define __ram_vector_table__            1
 
 #if (defined(__ram_vector_table__))
-  #define __ram_vector_table_size__    0x00000200
```

Comment:
Nope, that change is not required. It just saves space. The commit "Update ram vector size to save space" just saves space and could be removed entirely. Let me know if you want me to.